### ISSUE TITLE
Tape MileStone

### ISSUE THREAD

**Issue by tonyyang-svail** [PaddlePaddle/tape#10]
- [ ] 1. Vision Net benchmark
   - [x] 1. Move to a new repo, PaddlePaddle/Tape (tony)
   - [x] 2. Implement loss->value(), very useful for debugging (kexin) #11 
   - [x] 3. Expose Operators
      - [x] 1. Xavier initializer (tony)
      - [x] 2. Reader (kexin)
      - [x] 3. softmax, mul, conv2d (tony)
      - [x] 4. dropout, pool2d, batch_norm (kexin)
      - [x] 5. relu, use something like F.relu (tony)
      - [x] 6. enable default attribute (kexin)
   - [x] 4. Test Backward (3 days, tony) #38
      - [x] 1. Write a graph drawer to print GraphViz #39
   - [x] 5. Test Adam optimizer (2 days, kexin) #36 
   - [x] 6. Save/Load Model ( 1 day) #54
      - [x] Add `Parameter` derived from `Variable` #44 
      - [x] Add `ParameterCollection` #44
      - [x] Expose Save and Load #47
   - [x] 7. Inference (1 day, kexin) #40 
      - [x] 1. Accuracy
   - [x] 8. GPU (2 day) #45
      - [x] Code clean up `CPUPlace`
      - [x] Reader
      - [x] `<< Variable`
   - [ ] 9. Benchmark benchmark/vgg
      - [x] DoubleBuffer
      - [ ] Without ScopeWrapper, direct InferShape and Operator::Run
- [ ] 2. Control Flow demo
   - [ ] 1. if/else
   - [ ] 2. while/for
   - [ ] 3. Nested while/for
      - [ ] 1. nested 
   - [ ] 4. GAN demo 
   - [ ] 5. Imitation demo
- [ ] 3. Mobile Deployment
   - [ ] 1. Support training
   - [ ] 2. Prune binary file
